test/bugs-reported/bug7.chpl

```diff
@@ -0,0 +1,15 @@
+use Graph;
```

Comment:
Created a JIRA issue available here: https://chapel.atlassian.net/projects/CHAPEL/issues/CHAPEL-199

Removing this as a "future" test.